Sources/App/Common/Extensions/HTTP/Response+Error.swift

```diff
@@ -0,0 +1,12 @@
+import Vapor
```

Comment:
If we use `Abort` maybe delete this file?

Reply:
done